src/utils.js

```diff
@@ -235,3 +235,30 @@ export function apiToMessage(string) {
 export function isBrowserNamespace(string) {
   return ['browser', 'chrome'].includes(string);
 }
+
+
+export function parseCspPolicy(policy) {
```

Comment:
Yeah the ones I saw seems quite basic. The only benefit would be that they might benefit from updates - since the spec changes quite a bit that might be worth something.